### PR TITLE
Docs: edit "Utility functions" & make "Debugger integration" toplevel

### PR DESCRIPTION
**Pull request by Jasha10**
Minor edits to the `OmegaConf.to_container` section of the documentation.

### CODE REVIEW

**Comment by omry** [comment]
Let's hold this back until we land #606 (which touches this doc too).

**Comment by Jasha10** [comment]
I've rebased against master now that #606 is closed.

**Review by omry — APPROVED** [review]
lgtm.

One thing:
The order of the functions right now does not represent their importance.
What do you changing the order to:
```
OmegaConf.to_container
OmegaConf.select
OmegaConf.update
OmegaConf.masked_copy
OmegaConf.is_missing
OmegaConf.is_interpolation
OmegaConf.is_none
OmegaConf.{is_config, is_dict, is_list}
```

?

**Comment by Jasha10** [comment]
> One thing:
> The order of the functions right now does not represent their importance.
> What do you changing the order to:
> 
> ```
> OmegaConf.to_container
> OmegaConf.select
> OmegaConf.update
> OmegaConf.masked_copy
> OmegaConf.is_missing
> OmegaConf.is_interpolation
> OmegaConf.is_none
> OmegaConf.{is_config, is_dict, is_list}
> ```
> 
> ?

Sounds good to me. I would vote to put `update` before `select`.
```
OmegaConf.to_container
OmegaConf.update
OmegaConf.select
OmegaConf.masked_copy
OmegaConf.is_missing
OmegaConf.is_interpolation
OmegaConf.{is_config, is_dict, is_list}
```

**Comment by omry** [comment]
I think select is more useful than update (at least I have been using it much more frequently).

**Comment by Jasha10** [comment]
Fair enough.

**Comment by Jasha10** [comment]
Done in https://github.com/omry/omegaconf/pull/614/commits/29cb357b0cc9a21bd30e0db9ec47ad1eb059f472.

File: docs/source/usage.rst
Comment (omry) [inline docs/source/usage.rst:884]: While you are at it, can you make debugger integration into a top level item? right now it's under misc functions.
Otherwise looks good.

**Review by omry — APPROVED** [review]

File: docs/source/usage.rst
Comment (Jasha10) [reply docs/source/usage.rst]: Done in https://github.com/omry/omegaconf/pull/614/commits/5c64c828def8d98fa3d6d6a6b170e1f278c57e1d

Comment (omry) [reply docs/source/usage.rst]: All good, thanks!

**Pull request merged**
